backend/migrations/sqls/20241002111606-init-schema-up.sql

```diff
@@ -15,6 +15,13 @@ CREATE TABLE transaction_results (
   transaction_result jsonb,
   head_cid TEXT
 );
+-- Users
+CREATE TABLE users (
+  oauth_provider TEXT,
+  oauth_user_id TEXT,
+  handle TEXT,
```

Comment:
There does not appear to be any constraint to `handle`, I'm assuming we want that column to be unique

Reply:
Thanks for the catch